src/common/Selects/ContactStatusSelect/ContactStatusSelect.tsx

```diff
@@ -0,0 +1,31 @@
+import { ListSubheader, MenuItem, Select, SelectProps } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useApiConstants } from 'src/components/Constants/UseApiConstants';
+import { useLocalizedConstants } from 'src/hooks/useLocalizedConstants';
+
+type ContactStatusSelectProps = Partial<SelectProps> & {
```

Comment:
It turns out that all the fields in `SelectProps` are optional, so we don't need `Partial`.

```suggestion
type ContactStatusSelectProps = SelectProps & {
```